### ISSUE TITLE
Feature/convert_heic_to_jpeg_quality

### ISSUE THREAD

**Issue by githubkab** [boredazfcuk/docker-icloudpd#51]
I really like your tool, especially the automatic conversion of heic to jpeg. Although, I noticed differences in file size between pictures converted by macos photos and your script. Likely due to level of compressibility. I was thinking it might be nice to add a feature to adjust quality vs file size? I prefer high quality over compression, others might not.

You are you using the [libheif](https://github.com/strukturag/libheif) package, right?

**Comment by boredazfcuk** [comment]
Thanks for bringing this to my attention. The libheif documentation doesn't detail the default quality, so I presumed it would be the highest available. Looking at the source code for libheif, it appears to only be 90 out of 100.

I too prefer quality, but I've set the default quality to 90 so that it mimics the default behaviour of the libheif library. Also, so that the conversion feature continues to operate at the compression level for users that have the feature enabled already.

Just re-create your container with an added variable **jpeg_quality** and set it to **100** so that your files are created at maximum quality.

I've pushed a new version of the container to Dockerhub. Let me know if you have any problems.
